packages/front-end/components/Admin/EditOrganization.tsx

```diff
@@ -96,6 +177,63 @@ const EditOrganization: FC<{
             onChange={(e) => setOwner(e.target.value)}
           />
         </div>
+        <div className="mt-3">
+          Verified Domain
+          <input
+            type="text"
+            className="form-control"
+            value={verifiedDomain}
+            onChange={(e) => setVerifiedDomain(e.target.value)}
+          />
+          <span className="text-muted small">
+            This is used so new users can auto join this org by matching email
+            domain.
+          </span>
+        </div>
+        <div className="mt-3">
+          Auto approve members with email domain
+          <Toggle
+            className="ml-2"
+            id="autoApproveMembers"
+            value={autoApproveMembers}
+            setValue={setAutoApproveMembers}
+          />
+        </div>
+        {isCloud() ? (
+          <>
+            <div className="mt-3">
+              Free Seats
+              <input
+                type="number"
+                min={0}
+                className="form-control"
+                value={freeSeats}
+                onChange={(e) => setFreeSeats(parseInt(e.target.value))}
+              />
+              <div>
+                <span className="text-muted small">
+                  Number of seats that will not be billed
+                </span>
+              </div>
+            </div>
+            <div className="mt-3">
+              Enable Enterprise
+              <Toggle
+                className="ml-2"
+                id="legacyEnterpriseToggle"
+                value={legacyEnterprise}
+                setValue={setLegacyEnterprise}
+              />
+              <div>
+                <span className="text-muted small">
+                  Organizations with enterprise enabled this way are not billed,
+                  and will not expire. This is a legacy feature. Be sure to also
+                  adjust the free seats to match org requirements.
```

Comment:
`free seats` does nothing in terms of limiting the number of users in the account.  I would just remove that language.  I would include deprecated language here.
```
<b>Deprecated:</b> This is an old way to enable enterprise features for an organization, which doesn't expire, and doesn't restrict seats. Please uncheck this and instead user Retool and set a licenseKey instead.
```

Reply:
done